Program.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.

```

Comment:
wrong newline?

Reply:
as discussed, there was a byte order mark at the beginning. This change somehow wasn't intended when i trimmed the whitespace as HoLLy suggested.